compiler/next/include/chpl/queries/stringify-functions.h

```diff
@@ -239,6 +243,24 @@ void operator()(std::ostream &stringOut,
 }
 };
 
+template<> struct stringify<long long int> {
+void operator()(std::ostream &stringOut,
```

Comment:
This template seems duplicated a lot. I wonder if we could just make the generic template fallback to using std::to_string rather than erroring out, but I'm guessing this was done on purpose?

You may want to double check that we're not missing any other common integral\floating point datatype.

Anyway, your changes certainly look right to me and I wouldn't hold off pushing a fix for a smoke-test failure.

Reply:
Not sure if we ever use them, but taking a quick look I think you're also missing: short int, unsigned short int, and long double.

Reply:
There's a table of fundamental datatypes here: https://en.cppreference.com/w/cpp/language/types
For integers I think we'd only need to cover the ones under the "Equivalent type" column.

If you really want to be thorough I'd check to see if the stuff under "Fixed width integer types" here also works: https://en.cppreference.com/w/cpp/types/integer, though I'm guessing maybe they'll coerce to the types you've already used in the templates?

Reply:
I don't like the duplicated code at all, I am creating a task to generalize the integral types. It kind of blew up into all those specializations...
Thanks for the links, I added all the types in the "Equivalent type" column and the in the floating point section, just in case.